vignettes/Demography.Rmd

```diff
@@ -199,48 +181,44 @@ dyn_dem_params <- set_demography(
   timesteps = c(0, 2*365),
   deathrates = matrix(c(deathrates, deathrates_increased), nrow = 2, byrow = TRUE)
 )
-
 ```
 
-## Simulation
+### Simulation
 Having established the parameter list for the dynamic death rate simulation, we can now run the simulation using the `run_simulation()` function.
 
 ```{r}
 # Run the simulation for the dynamic death rate demography
 dyn_dem_output <- run_simulation(sim_length, dyn_dem_params)
-
 ```
 
-## Visualisation
+### Visualisation
 
 Bar plots of the age structure of the human population at the end of each year show the marked effect that increasing the death rate within a small subset of age classes can have on the demography of the human population over time.
 
-```{r, fig.width = 7.2, fig.height = 3, fig.fullwidth = TRUE}
-
+```{r, fig.align = 'center', out.width='100%', fig.asp=0.35}
```

Comment:
The figures in this file render very blurry for me, with large text and small plots. Is this because of the aspect changes?

Reply:
It's not blurry for me

Reply:
Same as above